.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,58 +4,43 @@ about: Create a report to help us improve existing features
 
 ---
 
-<!--
-DELETE ME
+# Checklist
+<!-- Place an x in the boxes to tick them: [x] -->
 
-Use the search function to see if someone else has already submitted the same bug report.
+- [ ] I have used the search function to see if someone else has already submitted the same bug report.
+- [ ] I will describe the problem with as much detail as possible.
+- [ ] If the bug only to occurs with a certain podcast, I will include the URL of that podcast.
 
-Try to describe the problem with as much detail as possible.
-
-Some bugs may only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`), as well as which version of AntennaPod you are using.
-
-If the bug only seems to occur with a certain podcast, please include the URL of that podcast.
-
-If possible, add instructions on how to reproduce the bug.
-
-If possible, add a logfile to your post. This is especially useful if the bug makes the application crash. AntennaPod has an `export logs` feature for this.
-
-Usually, you can take a screenshot of your smartphone by pressing *Power* + *Volume down* for a few seconds.
-
-You don't need to adhere to the template strictly. Feel free to leave out information you feel is not important or does not make sense.
-
-If you are experiencing a crash, including the stacktrace will likely get it fixed sooner.
--->
-
-<!-- READ THIS: The latest version may be different depending on your device. You can find the version in AntennaPod's settings. -->
-**App version**: x.y.z (state whether from Google Play/F-Droid/Custom built APK)
+# System info
+<!-- The following information is very important to fill out because some bugs may only occur on certain devices or versions of Android. -->
 
+**App version**: x.y.z (Where did you download the app?)
+<!-- The latest version may be different depending on your device. You can find the version in AntennaPod's settings. -->
 
 **Android version**: 5.x (Please mention if you are using a custom rom!)
 
-
 **Device model**:
 
-
-**Expected behaviour**:
-
-
-**Current behaviour**:
-
-
-**First occurred**: (e.g. about x days/weeks ago)
-
+# Bug description
 
```

Comment:
I would start letter the user explain first what the problem is, then ask more difficult questions (for which they need to look up the answer). So:
- Bug description
  - Describe the problem in 1 short sentence
  - URL of affected podcast
  - Steps to reproduce
  - Expected behaviour
  - Current behaviour
  - First occurred
- "The context"
  - Environment
  - App version
  - Android version
  - Device model
- "Additional info"
  - "Crash logs" (if we want regular users to share the logs from the app, then we should make the title less technical)
  - Screenshots ("Is there something off in the interface? A screenshot may help.")